Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -448,11 +438,11 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
     bool m_fullySynchronousModeIsAllowedForTesting { false };
     bool m_ignoreTimeoutsForTesting { false };
     bool m_didReceiveInvalidMessage { false };
+    int m_incomingMessagesThrottlingLevel { -1 };
```

Comment:
I think we're trying to move away from magic values in WebKit. This may look nicer as an std::optional<unsigned>.